ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJBTransformers.java

```diff
@@ -301,4 +321,70 @@ private static void registerPassivationStoreTransformers_1_2_1_and_1_3_0(Resourc
                 .setValueConverter(AttributeConverter.Factory.createHardCoded(new ModelNode().set(TimeUnit.SECONDS.name()), true), EJB3SubsystemModel.IDLE_TIMEOUT_UNIT)
         ;
     }
+
+    /**
+     * This Combined Transformer manages this transformation from EAP7 to previous versions (attribute definition/xml markup):
+     *
+     * DEFAULT_SFSB_CACHE / cache-ref --> DEFAULT_CLUSTERED_SFSB_CACHE / clustered-cache-ref
+     * DEFAULT_SFSB_PASSIVATION_DISABLED_CACHE / passivation-disabled-cache-ref --> DEFAULT_SFSB_CACHE / cache-ref
+     *
+     */
+    private static class SfsbChacheRefTransformer implements CombinedTransformer {
+        private final Map<String, String> renames;
+
+        public SfsbChacheRefTransformer(){
+            renames = new HashMap<>();
+            renames.put(DEFAULT_SFSB_CACHE, EJB3SubsystemModel.DEFAULT_CLUSTERED_SFSB_CACHE);
+            renames.put(DEFAULT_SFSB_PASSIVATION_DISABLED_CACHE, EJB3SubsystemModel.DEFAULT_SFSB_CACHE);
+        }
+
+        @Override
+        public TransformedOperation transformOperation(TransformationContext context, PathAddress address, ModelNode operation) throws OperationFailedException {
+            if (operation != null && !(operation.hasDefined(OPERATION_HEADERS) && operation.get(OPERATION_HEADERS, "push-to-servers").asBoolean(false)) ){
+                String originalAttribute = Operations.getAttributeName(operation);
+                if (renames.containsKey(originalAttribute)){
+                    operation = operation.clone();
+                    operation.get(NAME).set(renames.get(originalAttribute));
+                }
+            }
+
+            return new TransformedOperation(operation, OperationResultTransformer.ORIGINAL_RESULT);
+        }
+
+        @Override
+        public void transformResource(ResourceTransformationContext context, PathAddress address, Resource resource) throws OperationFailedException {
+            Resource untransformedResource = context.readResource(PathAddress.EMPTY_ADDRESS);
+            ModelNode untransformedModel = Resource.Tools.readModel(untransformedResource);
+
+            String defaultSfsbCache = untransformedModel.get(DEFAULT_SFSB_CACHE).asString();
+            String defaultSfsbPassivationDisabledCache = untransformedModel.get(DEFAULT_SFSB_PASSIVATION_DISABLED_CACHE).asString();
+
+            ModelNode transformed = resource.getModel();
+            transformed.get(DEFAULT_SFSB_CACHE).set(defaultSfsbPassivationDisabledCache);
+            transformed.get(EJB3SubsystemModel.DEFAULT_CLUSTERED_SFSB_CACHE).set(defaultSfsbCache);
+            transformed.remove(DEFAULT_SFSB_PASSIVATION_DISABLED_CACHE);
+
+            final ResourceTransformationContext childContext = context.addTransformedResource(PathAddress.EMPTY_ADDRESS, resource);
+            childContext.processChildren(resource);
+        }
+    }
+
+    private static class AddSfsbChacheRefTransformer implements OperationTransformer{
```

Comment:
Same "chache" spelling problem.  Kinda reminds me of ehcache. :)

Reply:
@pferraro I've fixed the spelling problem.